webapp/views/layout.html

```diff
@@ -168,6 +171,7 @@
                       <li><a href="{{= URL('about','materials-and-methods') }}">Materials and methods</a></li>
                       -->
                       <li><a href="{{= URL('about','references') }}">Bibliographic references</a></li>
+                      <li><a href="{{= URL('about', 'statistics') }}">Statistics</a></li>
```

Comment:
This is mooted by the institution of the 'development' branch
https://docs.google.com/document/d/1k1NJhnE4I6Klw7u9RTpbrBgqfhWkvoR8N0fogQ29rEA/edit
.

So please do *not* remove those from the menu in this branch.

On Thu, Jan 8, 2015 at 2:12 PM, Jim Allman <notifications@github.com> wrote:

> In webapp/views/layout.html
> <https://github.com/OpenTreeOfLife/opentree/pull/495#discussion-diff-22673568>
> :
>
> > @@ -168,6 +171,7 @@
> >                        <li><a href="{{= URL('about','materials-and-methods') }}">Materials and methods</a></li>
> >                        -->
> >                        <li><a href="{{= URL('about','references') }}">Bibliographic references</a></li>
> > +                      <li><a href="{{= URL('about', 'statistics') }}">Statistics</a></li>
>
> Let's remove this for now, so we can review the page on Production.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenTreeOfLife/opentree/pull/495/files#r22673568>.
>